App.tsx

```diff
@@ -1,20 +1,38 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { SafeAreaView, StatusBar, StyleSheet, Text, View } from 'react-native';
```

Comment:
Library imports on top, a space and then rest imports

Reply:
Resolved